Source/Scene/Model.js

```diff
@@ -2822,14 +2983,30 @@ define([
             this._boundingSphere = computeBoundingSphere(this.gltf);
             this._initialRadius = this._boundingSphere.radius;
 
-            var extensions = this.gltf.extensions;
-            if (defined(extensions) && defined(extensions.CESIUM_RTC)) {
-                this._rtcCenter = Cartesian3.fromArray(extensions.CESIUM_RTC.center);
-                this._rtcCenterEye = new Cartesian3();
+            var extensionsUsed = this.gltf.extensionsUsed;
```

Comment:
In order to keep `Model.update` reasonable-ish, can we move this check to a separate function that is called from here?